### ISSUE TITLE
Display Help for Value Options

### ISSUE THREAD

**Issue by austinwagner** [gsscoder/commandline#193]
The automatic help text builder ignores positional arguments and there is no way to provide help text for them. If I have this definition:

    [Verb("dosomething")]
    class DoSomethingOptions
    {
        [Value(0, Required = True, HelpText = "Number of foos")]
        public string Foo { get; set; }

        [Value(1, Required = False, HelpText = "Name of the bar")]
        public string Bar { get; set; }
    }

When I run `app.exe help dosomething`, I expect to see something similar to:

    Usage: app.exe dosomething FOO [BAR]
    
    FOO          Number of foos
    BAR          (Optional) Name of the bar

As it is now, there is no way to know that those positional arguments exist.

**Comment by gsscoder** [comment]
@austinwagner, this problem was addressed by #162 (and similar issue).

I've fixed this issue yesterday, certainly before you posted. At the moment the output did not print if something is optional or not. The class you defined will generate the following help screen:
```bash
value pos. 0         Number of foos
value pos. 1         Name of the bar
```
Now you can define a name for this type of option using `MetaName`:
```csharp
class DoSomethingOptions {
    [Value(0, Required = True, MetaName="foo", HelpText = "Number of foos")]
    public string Foo { get; set; }

    [Value(1, Required = False, MetaName="bar", HelpText = "Name of the bar")]
    public string Bar { get; set; }
}
```
Will result in:
```bash
foo (pos. 0)          Number of foos
bar (pos. 1)          Name of the bar
```

This is the current status, if you want/need something different or more control over output, I suggest open a new specific issue.

Thanks you again for posting here.

**Comment by gsscoder** [comment]
Since this is implemented, I close the issue.
:smile:
